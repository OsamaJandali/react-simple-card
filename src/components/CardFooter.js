// @flow

import React from "react";

type CardFooterProps = {
	children: any
};

const styles = {
	display: "flex",
  alignItems: "center",
	padding: "10px 18px",
	borderTop: "1px solid #ccc",
	marginTop: "0"
}

export default ({ children }: CardFooterProps) => (
	<div className="card__footer" style={styles}>{children}</div>
);
