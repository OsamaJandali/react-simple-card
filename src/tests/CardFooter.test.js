import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import CardFooter from "../components/CardFooter";

it("renders CardFooter correctly", () => {
	const wrapper = shallow(
		<CardFooter>
			Vestibulum mollis elit nec leo venenatis, nec elementum nunc.
		</CardFooter>
	);
	expect(toJson(wrapper)).toMatchSnapshot();
});
