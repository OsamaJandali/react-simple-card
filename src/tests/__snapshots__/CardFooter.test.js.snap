// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CardFooter correctly 1`] = `
<div
  className="card__footer"
>
  Vestibulum mollis elit nec leo venenatis, nec elementum nunc.
</div>
`;
